examples/mxnet/gcmc/README.md

```diff
@@ -0,0 +1,58 @@
+# Graph Convolutional Matrix Completion
+
+Paper link: [https://arxiv.org/abs/1706.02263](https://arxiv.org/abs/1706.02263)
+Author's code: [https://github.com/riannevdberg/gc-mc](https://github.com/riannevdberg/gc-mc)
+
+The implementation does not handle side-channel features and mini-epoching and thus achieves
+slightly worse performance when using node features.
+
+Credit: Jiani Zhang ([@jennyzhang0215](https://github.com/jennyzhang0215))
+
+## Dependencies
+* MXNet 1.5.0+
+* pandas
+* gluonnlp
+
+## Data
+
+Download the MovieLens datasets and unzip into `data_set/ml-?` directory.
```

Comment:
Why not move the dataset to our S3 bucket and let the program handle downloading, unzipping, etc.?

Reply:
Done